deploy/helm/clickhouse-operator/README.md.gotmpl

```diff
@@ -0,0 +1,17 @@
+{{ template "chart.header" . }}
```

Comment:
could you explain, why do we need this file?
it will help for helm-docs to generate README.md?

Reply:
Please see the PRs description